<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiskGui_Catchment_Scenario_Dia</class>
 <widget class="QDialog" name="RiskGui_Catchment_Scenario_Dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select scenarios for calculation of the catchment area risk...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc">
    <normaloff>:/risk_icon</normaloff>:/risk_icon</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="RiskGui_Catchment_Scenario_Table_Wid" name="tableWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>209</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RiskGui_Catchment_Scenario_Table_Wid</class>
   <extends>QTableWidget</extends>
   <header>RiskGui_Catchment_Scenario_Table_Wid.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc"/>
 </resources>
 <connections/>
</ui>
